actix-http/src/header/shared/mod.rs

```diff
@@ -1,13 +1,15 @@
 //! Originally taken from `hyper::header::shared`.
 
 mod charset;
+mod content_encoding;
 mod encoding;
 mod entity;
 mod extended;
 mod httpdate;
 mod quality_item;
 
 pub use self::charset::Charset;
+pub use self::content_encoding::ContentEncoding;
```

Comment:
whats the issue with ContentEncoding?

Reply:
It is exposed in https public api for the encoder and decoder.